protos/geofence/geofence.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package mavsdk.rpc.geofence;
+
+option java_package = "io.mavlink.mavsdk.geofence";
+option java_outer_classname = "GeofenceProto";
+
+service GeofenceService {
+    rpc SendGeofence(SendGeofenceRequest) returns(SendGeofenceResponse) {}
+}
+
+
+message Polygon {
+    message Point {
+        double latitude_deg = 1;
+        double longitude_deg = 2;
+    }
+
+    enum Type {
+        INCLUSION = 0;
+        EXCLUSION = 1;
+    }
+
+    repeated Point points = 1;
+    Type type = 2;
+}
+    
+message SendGeofenceRequest {
```

Comment:
Can we name this consistent with the Mission proto? So we also use `upload` and `download? @JonasVautherin what do you think?

Reply:
Good catch! I agree!

Reply:
@julianoes So to clarify, you would like me to change `SendGeofenceRequest` to `UploadGeofenceRequest`? Also, I more or less copied the names from the C++ core - would you like me to look into changing that too?

Reply:
Ah sorry, I can see how that is confusing. Yes, let's do `Upload`.

Reply:
I updated the proto file. Sorry for a couple of messy commits, I ran into merge conflicts since I forgot that some typos were fixed online.